components/hero-home.tsx

```diff
@@ -93,18 +92,11 @@ export default function HeroHome() {
                     href="#0"
                   >
                     <span className="relative inline-flex items-center">
-                      Start Free Trial{" "}
-                      <span className="ml-1 tracking-normal text-blue-300 transition-transform group-hover:translate-x-0.5">
-                        -&gt;
-                      </span>
+                      Join Waitlist{" "}
```

Comment:
Good job moving ahead of the task. But do make sure your communicate with project manager or product owner (who manage the kanban board) that this out of scope task is also included in this PR. Usually we want to keep changes matching scope